common/predictive-text/.build-builder

```diff
@@ -0,0 +1,4 @@
+The presence of this file tells CI to use the new builder_ style parameters for build.sh.
+
+Once all branches for 16.0+ are updated to merge the chore/web/more-script-updates branch, then we
+can remove this file and the corresponding bash test branches in CI.
```

Comment:
Ah, so we're not (currently) planning to make a separate config like with `/core/`?  The version you referenced in an earlier comment contained text along these lines, so I just made the text more appropriate to this location.

Reply:
> Ah, so we're not (currently) planning to make a separate 16.0+ config like appears to be planned for `/core/`?

The only 16.0 config for core is the WASM test build -- because it is new in 16.0. We are trying not to replace existing CI configurations, rather just tweaking them to support new versions as needed. The version number shown in the CI titles is a little confusing, I agree, and we'll probably drop them in future maintenance of CI (once I go ahead and delete 13.0 configs, probably.)

Reply:
> The file you referenced in an earlier comment (or, at least the version I found) contained text along these lines, so I just made the text more appropriate to this location.

Yes, that was probably wrong on my part, sorry!

> ... or is it that the same branch that results in the 'outer' if-else would effectively have the same condition as the inner if-else? That... makes a bit more sense, now that I think about it.

And ... you've lost me here!

Reply:
From the CI config:

```
rem Keyman 16 moves Keyman Core to core/
echo Check for core/build.sh [Keyman 16+]
if exist core\build.sh (
  cd core
  if exist .build-builder (
    "C:\Program Files\Git\bin\bash.exe" --init-file "c:\Program Files\Git\etc\profile" -l ./build.sh configure:x86 build:x86 test:x86 configure:x64 build:x64 test:x64
  ) else (
	rem TODO: this branch not required once chore/core/7247-use-builder-for-build-sh changes merged into all branches on 16.0+
    "C:\Program Files\Git\bin\bash.exe" --init-file "c:\Program Files\Git\etc\profile" -l ./build.sh
  )
)
```

Once we've fully merged the referenced changeset into all remaining 16.0+ branches, `if exist core\build.sh` and `if exist .build-builder` will (effectively) be equivalent checks.  So, chalk it up to a "right, for the wrong reasons" re: the contents of the `.build-builder` file - we'll be able to simplify the quoted part of the CI config above.

Reply:
For core, yes, we can simplify due to the move from /common/core/desktop/ to /core/, but for other places, we probably can't simplify in the same way -- because build.sh already existed in the same place for 15.0.